content/en/docs/azure/deploy/install-kubeflow.md

```diff
@@ -135,7 +145,7 @@ Run the following commands to set up and deploy Kubeflow.
 
 1. Open Kubeflow Dashboard
 
-The default installation does not create an external endpoint but you can use port-forwarding to visit your cluster. Run the following command and visit http://localhost:8080.
+The default installation does not create an external endpoint but you can use port-forwarding to visit your cluster. Run the following command and visit <http://localhost:8080>.
```

Comment:
Line 148: surround the site name with back ticks instead of <>:

```suggestion
The default installation does not create an external endpoint but you can use port-forwarding to visit your cluster. Run the following command and visit `http://localhost:8080`.
```

Reply:
fixed